test.nix

```diff
@@ -57,6 +57,7 @@ in
         bottom = ctpEnable;
         btop = ctpEnable;
         fish = ctpEnable;
+        git.delta = ctpEnable;
```

Comment:
git also needs to be enabled for this to be applied - otherwise this works great. thanks!

Reply:
Git is now enabled in test.nix